plugin.video.videodevil/resources/settings.xml

```diff
@@ -1,10 +1,45 @@
 <settings>
-   <setting id="hide_warning" type="bool" label="30002" default="false"/>
-   <setting id="enable_debug" type="bool" label="20191" default="false"/>
-   <setting type="sep"/>
-   <setting id="video_type" type="enum" lvalues="30013|30014|30015|30016" label="30012" default="0"/>
-   <setting type="sep" />
-   <setting id="download" type="bool" label="30007" default="false"/>
-   <setting id="download_path" type="folder" enable="eq(-1,true)" source="video" label="30008"  option="writeable" default=""/>
-   <setting id="download_ask" type="bool" enable="eq(-2,false)" label="30009" default="false"/>
+  <category label="30218">
+    <setting id="hide_warning" type="bool" label="30002" default="false"/>
+    <setting id="enable_debug" type="bool" label="20191" default="false"/>
+    <setting type="sep"/>
+    <setting id="video_type" type="enum" lvalues="30013|30014|30015|30016" label="30012" default="0"/>
+    <setting type="sep" />
+    <setting id="download" type="bool" label="30007" default="false"/>
+    <setting id="download_path" type="folder" enable="eq(-1,true)" source="video" label="30008"  option="writeable" default=""/>
+    <setting id="download_ask" type="bool" enable="eq(-2,false)" label="30009" default="false"/>
+  </category>
+  <category label="Folder">
```

Comment:
We define our own.

Add 
<string id="30017">Enabled sites</string>
To plugin.video.videodevil/resources/language/English/strings.xml
And reference 30017 in this file